backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -61,45 +61,46 @@ public TestOrderService(
     _pls = pls;
   }
 
-  @AuthorizationConfiguration.RequirePermissionStartTest
-  public List<TestOrder> getQueue(String facilityId) {
-    Facility fac = _os.getFacilityInCurrentOrg(UUID.fromString(facilityId));
+  @AuthorizationConfiguration.RequirePermissionStartTestAtFacility
+  public List<TestOrder> getQueue(UUID facilityId) {
+    Facility fac = _os.getFacilityInCurrentOrg(facilityId);
     return _repo.fetchQueue(fac.getOrganization(), fac);
   }
 
   @Transactional(readOnly = true)
-  @AuthorizationConfiguration.RequirePermissionReadResultList
+  @AuthorizationConfiguration.RequirePermissionReadResultListAtFacility
   public List<TestEvent> getTestEventsResults(UUID facilityId, Date newerThanDate) {
     Facility fac = _os.getFacilityInCurrentOrg(facilityId); // org access is checked here
     return _terepo.getTestEventResults(
         fac.getInternalId(), (newerThanDate != null) ? newerThanDate : new Date(0));
   }
 
   @Transactional(readOnly = true)
-  @AuthorizationConfiguration.RequirePermissionReadResultList
-  public TestEvent getTestResult(UUID id) {
+  @AuthorizationConfiguration.RequirePermissionReadResultListForTestEvent
+  public TestEvent getTestResult(UUID testEventId) {
     Organization org = _os.getCurrentOrganization();
-    return _terepo.findByOrganizationAndInternalId(org, id);
+    return _terepo.findByOrganizationAndInternalId(org, testEventId);
   }
 
   @Transactional(readOnly = true)
-  @AuthorizationConfiguration.RequirePermissionReadResultList
+  @AuthorizationConfiguration.RequirePermissionReadResultListForPatient
   public List<TestEvent> getTestResults(Person patient) {
-    return _terepo.findAllByPatient(patient);
+    // NOTE: this may change. do we really want to limit visible test results to only
```

Comment:
@aliciabeckett-gov @katiealoisi @hmyers-cms would like your input here. are we sure, when loading the test results for a specific patient, we really only want to see the test results from the caller's accessible facilities? would the full history of the patient's test results constitute a medical history that the caller should have full access to when loading the patient?

Reply:
IMO we should load the past test results for that patient, even if the user doesn't have access to the facility with the result. The caller should have access to the latest test results if they have access to the patient.

Reply:
A question re: assigning users to facilities that I have, possibly because I wasn't privy to initial discussions on the data model:

Currently, when you create a patient, you select their facility (or specify "All Facilities). I can see how this is more performant (e.g., having fewer patients to search through when creating a test). But might it also cause confusion and/or data issues if a patient was created at Facility X and is now going for a test at Facility Y? A system user might think (a) this patient isn't in the system at all, so I should create a new record and (b) this patient has never been tested before.

I'm not sure how likely this scenario is at all, just a thought that crossed my mind.

Reply:
@aliciabeckett-gov to confirm, this function is to get all test results for a patient, not just the most recent. the most recent test will always be available to anyone who can see the patient. can you confirm that the caller should have access to _all_ previous tests of a patient (not just the most recent), regardless of facility?

Reply:
And @nickscialli-usds after this PR, a patient at Facility X indeed can't go for a test at Facility Y -- https://github.com/CDCgov/prime-simplereport/pull/1072/files#diff-ea6e46ce76b782b8c29dbe1ed4006220753b58756ba28063764398ca5f051007R167. I agree this could be a confusing user experience, though perhaps something to fix after this PR. Thoughts @hmyers-cms @aliciabeckett-gov @katiealoisi ?

Reply:
@jeremyzitomer-usds good call out! The entry only should only have access to the most recent test result for the person being tested.

Reply:
and what about a **_standard_** user? should a standard user with limited facility access still have access to all of a patient's past tests, regardless of facilities? (hoping selfishly that the the answer is no, because if the answer is yes i'd need to rewire things a bit.)

Reply:
They should have access to a patient's _**last**_ test result, regardless of facilities for the time of test questions.

However if this is going to cause a big refactor we can skip it, log a ticket, and deal with it if we get a complaint.

Reply:
got it. that is how the current system is designed and won't require a refactor :) thanks!